### PR TITLE
Update minimum browser versions

### PR DESCRIPTION
**Pull request by Gudahtt**
The minimum browser versions have been updated to Chromium v80 and Firefox v78.

~Chromium v78 is >2 years old, and very few users use older versions than that.~
Updated to v80 to provide support for optional chaining

Firefox v78 is the _previous_ extended support release. The current Firefox extended support release is v91.

Related: #6805

Manual testing steps:  
  - Ensure that the extension still works on versions equal to or above the new minimum versions.

### CODE REVIEW

**Review by hmalik88 — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

**Review by rekmarks — DISMISSED** [review]

**Comment by Gudahtt** [comment]
Apparently this might affect up to 1% of the userbase, though it's likely a bit lower than that. More details here: https://github.com/MetaMask/metamask-extension/issues/6805#issuecomment-979467319

**Comment by Gudahtt** [comment]
I'm marking this as "DO-NOT-MERGE" for now, to give those away on holiday the chance to weigh in about whether we should do this now, or whether we should wait until after we add more in-app messaging asking users to update.

**Comment by darkwing** [comment]
Idea that @Gudahtt and I were discussing was parsing the UA and, if the user is stuck on an old browser, displaying a persistent message on top of the home screen to let the user know they wont receive MM updates.  We'd need that to get merged the release before this gets merged.

**Comment by Gudahtt** [comment]
The issue #16283 has been created to track adding a notification warning users if we're about to drop support for their browser. That will unblock this PR.

File: app/manifest/v2/chrome.json
Comment (Gudahtt) [inline app/manifest/v2/chrome.json:7]: Should probably update this to at least 80, so that we can use the optional chaining operator.

File: babel.config.js
Comment (Gudahtt) [inline babel.config.js:8]:
```suggestion
      browsers: ['chrome >= 80', 'firefox >= 78'],
```

**Review by digiwand — DISMISSED** [review]
it appears the `DO-NOT-MERGE` tag was added for the last holiday season and can be removed now - LGTM! 🎄

**Comment by Gudahtt** [comment]
Thanks for the review @digiwand ! This was blocked by #16283, but is unblocked now that that has been included in the v10.26.0 RC.

**Review by FrederikBolding — APPROVED** [review]

**Review by digiwand — APPROVED** [review]

**Pull request merged**
